src/pages/news/news-detail/news-detail.style.css

```diff
@@ -0,0 +1,26 @@
+hr{
+    clear: both;
+}
+h3{
+    font-size: 1.2rem;
+}
+h2{
+    text-align: center;
+}
+iframe{
```

Comment:
Do we need this css styles? As I have seen `iframe` has been deleted from the component

Reply:
Yes. we need this style because we carn receive video files in an iframe inside of news text.